package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fiverr/eslint-config-fiverr",
   "moduleName": "eslint-config-fiverr",
-  "version": "0.0.9",
+  "version": "0.0.10",
```

Comment:
Because it is not backward compatible, small changes like this one actually qualify for a major.